src/yuzu/configuration/configure_motion_touch.cpp

```diff
@@ -187,13 +182,58 @@ void ConfigureMotionTouch::ConnectEvents() {
     });
 }
 
+void ConfigureMotionTouch::OnUDPAddServer() {
+    QRegExp re(tr("^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?["
+                  "0-9][0-9]?)$")); // a valid ip address
+    bool ok;
+    QString port_text = ui->udp_port->text();
+    QString server_text = ui->udp_server->text();
+    QString server_string = server_text + tr(":") + port_text;
```

Comment:
```suggestion
    const QString server_string = tr("%1:%2").arg(server_text, port_text);
```

Is the most localization friendly way of rendering this string.